react-app/src/app/components/Game/index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/jsx-key */
+/* eslint-disable no-use-before-define */
+import React, { Component } from 'react';
+
+import Board from '../Board';
+
+import './styles.css';
+
+class Game extends Component {
+  state = {
+    history: [{ squares: Array(9).fill(null) }],
+    xIsNext: true
+  };
+
+  handleClick(i) {
+    const history = this.state.history;
+    const current = history[history.length - 1];
+    const squares = current.squares.slice();
+    if (calculateWinner(squares) || squares[i]) return;
+    squares[i] = this.state.xIsNext ? 'X' : 'O';
+    this.setState({
+      history: history.concat([{ squares }]),
+      xIsNext: !this.state.xIsNext
+    });
+  }
+
+  render() {
+    const history = this.state.history;
+    const current = history[history.length - 1];
+    const winner = calculateWinner(current.squares);
+
+    let status;
+    if (winner) {
+      status = `Winner: ${winner}`;
+    } else {
+      status = `Next player: ${this.state.xIsNext ? 'X' : 'O'}`;
+    }
+
+    const moves = history.map((step, move) => {
+      const desc = move ? `Go to move # ${move}` : `Go to game start`;
+      return (
+        <li>
+          <button onClick={() => this.jumpTo(move)}>{desc}</button>
+        </li>
+      );
+    });
+
+    return (
+      <div className={'game'}>
+        <div className={'game-info'}>
+          <Board squares={current.squares} onClick={i => this.handleClick(i)} />
+          <div>{status}</div>
+          <ol>{moves}</ol>
+        </div>
+      </div>
```

Comment:
This component is getting pretty large, divide it into index.js to contain the logic and layout.js to contain the html. Then you can import here the layout and render it